scripts/build.sh

```diff
@@ -43,7 +43,7 @@ echo "==> Building..."
 gox \
     -os="${XC_OS}" \
     -arch="${XC_ARCH}" \
-    -ldflags "-X main.GitCommit ${GIT_COMMIT}${GIT_DIRTY}" \
+    -ldflags "-X=main.GitCommit ${GIT_COMMIT}${GIT_DIRTY}" \
```

Comment:
This has been fixed in master with: https://github.com/Yelp/terraform/commit/963a1b4f4b4ba6842203de9ed374676d8d001052